checks/md/md.go

```diff
@@ -0,0 +1,18 @@
+// Package md contains checks for markdown.
+package md
+
+import (
+	"github.com/russross/blackfriday"
+	"k8s.io/md-check/checks"
+)
+
+// ParseCheck ensures that
+func ParseCheck(rptr checks.ErrReporter, fname string, contents []byte) error {
+	// Check to make sure the markdown parses successfully, but don't do anything
+	// special
+	blackfriday.MarkdownBasic(contents)
```

Comment:
What does this do if it doesn't parse?  Panic?  Should we catch it and return an error instead?